bin/run.sh

```diff
@@ -1,4 +1,3 @@
-# turn off slack for now!
-#invoke notify
+# invoke notify || echo 'failed to start slack'
```

Comment:
I vote for doing this later. We need new login from slack, so maybe just wait and maybe we will just want to move that to FEC slack.